internal/concierge/server/server.go

```diff
@@ -181,6 +187,22 @@ func (a *App) runServer(ctx context.Context) error {
 		return fmt.Errorf("could not configure aggregated API server: %w", err)
 	}
 
+	oneDayInSeconds := int64(24 * 60 * 60)
+	k8sClient, err := kubeclient.New()
+	if err != nil {
+		return fmt.Errorf("could not create default kubernetes client: %w", err)
+	}
+	aggregatedAPIServerConfig.ExtraConfig.TokenClient = tokenclient.New(
+		podInfo.Namespace,
+		cfg.NamesConfig.ImpersonationProxyServiceAccount,
+		k8sClient.Kubernetes,
```

Comment:
It seems a little unfortunate that we are creating another kube client for this, although it might be fine. Note that it is not injecting our custom API suffix middleware, so if it were accidentally used by other code in the future to make other API calls (to any Pinniped API) then it may not work correctly. Perhaps we should only pass in the `ServiceAccount` portion of this client into the `token client.New` to ensure that the token client cannot make any other API calls using this client? Since that client would already be configured for the appropriate namespace, then we wouldn't need to pass the namespace to the constructor anymore.

Reply:
Maybe we should also add a comment here that we are purposefully not using leader election on this client, any why.

Reply:
Comment added, see next push.